src/test/kotlin/day03/HeroTest.kt

```
@@ -1,34 +1,34 @@
+ // 준비
+ var hero = Hero(name = "홍길동", hp = 10)
+
+ // 실행
+// hero.sleep()
+
+ // 확인
+ assertEquals(100, hero.hp)
+ }
+
+@org.junit.Test
+  fun run() {}
```

detekt (1.23.7)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)